rust/main/chains/hyperlane-sealevel/src/keypair.rs

```diff
@@ -0,0 +1,41 @@
+use solana_sdk::{signature::Keypair, signer::Signer};
+
+/// Wrapper around solana_sdk's Keypair.
+/// This implements a custom Debug so the private keys are
+/// not exposed.
+pub struct SealevelKeypair(pub Keypair);
```

Comment:
curious how you feel about Deref, iirc it's kinda controversial but it could maybe allow for a more ergonomic use of this type if we do a
```
impl Deref for SealevelKeypair {
  type Target = Keypair;
  
  fn deref(&self) -> &Self::Target {
    &self.keypair
  }
}
```

Reply:
Ah yeah, that works 👍